src/components/RecommendationPage.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import axiosInstance from '../apis/axiosClient';
+import { Typography, CircularProgress, List, ListItem } from '@mui/material';
+import { useAuth } from '@akosasante/react-auth-context';
+const RecommendationPage = () => {
+  const { user } = useAuth();
+  const userId = user ? user.userId : null;
+
+  const [recommendations, setRecommendations] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (userId) {
+      axiosInstance
+        .get(`/recommendations/${userId}`)
+        .then((response) => {
+          setRecommendations(response.data.recommendations);
+          setLoading(false);
+        })
+        .catch((error) => {
+          console.error('Error fetching recommendations:', error);
+          setError('An error occurred while fetching recommendations.');
+          setLoading(false);
+        });
+    } else {
+      setLoading(false);
+    }
+  }, [userId]);
+
+  return (
+    <div>
+      <Typography
+        fontWeight="bold"
+        color="text.secondary"
+        variant="h5"
+        spacing={0.5}
+        sx={{ mb: 2, p: 3 }}
+      >
+        Recommendations
+      </Typography>
+      {error ? (
+        <Typography color="error">{error}</Typography>
```

Comment:
The `color` attribute on the material-ui `Typography` component should map to a value from the Theme being used. 

This page shows all the values available (of course in our app some of these might have different values based on customization we've done): [https://mui.com/material-ui/customization/default-theme/#main-content](https://mui.com/material-ui/customization/default-theme/#main-content)

All the colors are under the `pallette` section. So `error` is not a color by itself, but rather an object with multiple options. So we need to pick one of those options, like for example we could make this:

```suggestion
        <Typography color="error.main">{error}</Typography>
```

and then the text will have the color "#d32f2f"

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/eb3a1c33-4c2e-4635-aeff-b22a6fa5839e)